### PR TITLE
Remove mentions of scatter

### PR DESCRIPTION
**Pull request by tristan957**
Scatter as a concept is being removed in HSE 3.0.

Signed-off-by: Tristan Partin <tpartin@micron.com>

### CODE REVIEW

**Review by alexttx — APPROVED** [review]

**Comment by tristan957** [comment]
Alex, I will let you merge this with after your other PR is merged.

**Pull request merged**
